.drone/drone.sh

```
@@ -0,0 +1,195 @@
+  fi
+
+  git clone https://github.com/boostorg/boost-ci.git boost-ci-cloned --depth 1
+  [ "$SELF" == "boost-ci" ] || cp -prf boost-ci-cloned/ci .
+  rm -rf boost-ci-cloned
+
+  if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      unset -f cd
+  fi
+
+  export BOOST_CI_TARGET_BRANCH="$TRAVIS_BRANCH"
+  export BOOST_CI_SRC_FOLDER=$(pwd)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)